lib/api.js

```diff
@@ -152,6 +152,23 @@ class Madge {
 		return graph.image(this.obj(), this.circular(), imagePath, this.config);
 	}
 
+	circularImage(imagePath) {
+		if (!imagePath) {
+			return Promise.reject(new Error('imagePath not provided'));
+		}
+
+		const circular = this.circular();
+
+		const newTree = Object.entries(this.obj())
+			.filter(([k, v]) => circular.some((x) => x.includes(k)))
```

Comment:
```suggestion
			.filter(([k]) => circular.some((x) => x.includes(k)))
```
Remove variable ```v``` as it's unused. The build is failing as this is causing the following linting error,
```error  'v' is defined but never used  no-unused-vars```